nats.go

```diff
@@ -61,6 +61,7 @@ const (
 	DefaultReconnectBufSize   = 8 * 1024 * 1024 // 8MB
 	RequestChanLen            = 8
 	DefaultDrainTimeout       = 30 * time.Second
+	DefaultFlusherTimeout     = 30 * time.Second
```

Comment:
maybe set to `time.Minute` that should be enough I think, and just before the default ping interval `2*time.Minute`